README.md

```diff
@@ -38,13 +38,31 @@ An [Elastic Email][] adapter for the [Bamboo][] email app for Elixir.
 
 4.  Follow the Bamboo [Getting Started Guide][getting_started].
 
-5.  To use ElasticEmail's [postBack][] functionality you can place a value in
+5.  To use [ElasticEmail's API parameters][email_send] you can place a value in
```

Comment:
A small thing here.

```markdown
5.  To use [Elastic Email's API parameters][email_send] that are not automatically
      handled by this plug-in natively, you can place a value in the `Email#private`
      parameter:
```

That should satisfy the documentation bit that not all parameters are supported by this feature, and why.